contents/blog/ab-testing-mistakes.md

```diff
@@ -0,0 +1,99 @@
+---
+title: 5 A/B testing mistakes and how to avoid them
+sidebar: Docs
+showTitle: true
+author: ['lior-neu-ner']
+date: 2023-04-24
+featuredImage: todo
+tags: ['experimentation']
+---
+
+Making a mistake in an A/B test can have disastrous implications. At best, it means you've wasted weeks waiting for results that are not useful. At worst, it can lead you to make the wrong product decisions. This blog post will explore five common A/B testing mistakes to avoid to ensure successful experiments!
+
+## 1. Not having a clear hypothesis
+
+Having a clear definition of what you are testing and why you are testing it will help you determine which metrics to measure and analyze. To understand this better, let's look at an example of a bad hypothesis:
+
+> Bad Hypothesis: Changing the color of the "Proceed to checkout" button will increase purchases.
+
+This is bad since it's unclear why we're testing this change and why we expect it to increase purchases. As a result, it's not clear what we need to measure in this experiment. Do we only need to count button clicks? Or is there something else we need to measure?
+
+Here's an improved version:
+
+> Good hypothesis: User research showed that users are unsure of how to proceed to the checkout page. Changing the button's color will lead to more users noticing it and thus more people will proceed to the checkout page. This will then lead to more purchases.
+
+It's now clear that we need to measure the following: 
+* Button clicks, to show if the change in color leads to more people noticing the button
+* Number of purchases, since more people arriving at the checkout page should mean more purchases
+
+This also makes it easier to investigate the experiment results, especially when they are not what we expected. For example:
+* If the color change didn't increase button clicks, there may be a different issue on the page.
+* If the number of button clicks increased but the number of purchases did not, there may be an issue with the checkout page.
+
+## 2. Not monitoring counter metrics
```

Comment:
```suggestion
## 2. Ignoring secondary metrics
```
We use this term in app.

Reply:
Counter metrics are different from secondary metrics (although they are a type of secondary metrics). Counter metrics measure negative effects specifically, while other secondary metrics can may also measure positive effects